WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataFunctional.php

```diff
@@ -25,17 +24,17 @@ public function load(ObjectManager $manager)
         $workflowFunctionValidator = new WorkflowFunction();
         $workflowFunctionValidator->addName($enName);
         $workflowFunctionValidator->addName($frName);
-        $workflowFunctionValidator->addRole($this->getReference('role-pending'));
-        $workflowFunctionValidator->addRole($this->getReference('role-published'));
-        $this->addReference('workflow_function-validator', $workflowFunctionValidator);
+        $workflowFunctionValidator->addRole($this->getReference('role-functional-draft-to-pending'));
+        $workflowFunctionValidator->addRole($this->getReference('role-functional-pending-to-published'));
+        $this->addReference('workflow_function_validator_functional', $workflowFunctionValidator);
 
         $enName = $this->generateTranslatedValue('en', 'Contributor');
         $frName = $this->generateTranslatedValue('fr', 'Contributeur');
         $workflowFunctionContributor = new WorkflowFunction();
         $workflowFunctionContributor->addName($enName);
         $workflowFunctionContributor->addName($frName);
-        $workflowFunctionContributor->addRole($this->getReference('role-draft'));
-        $this->addReference('workflow_function-contributor', $workflowFunctionContributor);
+        $workflowFunctionContributor->addRole($this->getReference('role-functional-published-to-draft'));
+        $this->addReference('workflow_function_contributor_functional', $workflowFunctionContributor);
```

Comment:
I'm not a big fan of mixing _ and - in the ref, you should choose one model and put it everywhere (maybe -)